### DISCUSSION TITLE
Comparison with other models

### DISCUSSION THREAD

**Discussion by dracar2s** [bingogome/samm#42] [General]
Hi. I was waiting for someone to implement SAM in medical imaging, and thank you for your interesting work. I skimmed through your paper and noticed that you compared SAMM to SimpleClick and RITM. 
May I ask how you used these models for DICOM images? I can’t seem to find any Slicer plugin or other software except a paid online service. 
It would be great if it is also implemented in your plugin.

**Comment by bingogome** [comment]
We did not make any substantial comparisons to other models, but more looked into the realtime-ness of modified SAM in here.
